api/src/OpenTelemetry/Internal/Common/Types.hs

```diff
@@ -79,3 +81,8 @@ data FlushResult
     -- unexported spans or @LogRecord@s.
     FlushError
   deriving (Show)
+
+
+data ExportResult
```

Comment:
Moved here because it should be common to `SpanExporter`s and `LogRecordExporter`s.